### DISCUSSION TITLE
Sabnzd LXC Access NFS Storage

### DISCUSSION THREAD

**Discussion by emddummi** [tteck/Proxmox#1751] [Help Needed]
Hi,

I installed Sabnzd as privileged Container and installed nfs-common.
I have Mount the drive and can See all files with ls - l

I can create New files with mkdir in console.

But if i want to use the directory in sabnzd it tells me that sabnzd has no Access to the directory

**Comment by LoadingStill** [comment]
What is the output of `ls -l`?

I am more asking what the first part is
la -la acroread
-rwxr-xr-x 1 10490 floppy 17242 May  8  2013 acroread
|----------| that part

from your container running SABnzbd
